web/packages/design/src/StepSlider/StepSlider.tsx

```diff
@@ -328,6 +329,14 @@ type Props<Flows> =
          * Optional if there is only one flow.
          */
         onSwitchFlow?(flow: keyof Flows): void;
+        /**
+         * defaultStepIndex is the step that will be shown on the first render, similar to
+         * defaultValue passed to the input tag.
+         *
+         * Since this value is used only for the initial render, it won't be persisted when
+         * switching flows – this will result in the current step index being reset to 0.
+         */
+        defaultStepIndex?: number;
```

Comment:
would `initStepIndex` or `initialStepIndex` be a better name? (up to you)

Reply:
I feel like the `default` suffix draws a stronger parallel to `defaultValue` from `<input>`.